packages/devtools_app/test/integration_test/in_file_args_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../integration_test/test_infra/run/_in_file_args.dart';
+
+const _testAppPath = 'test/test_infra/fixtures/memory_app';
+
+late final _defaultArgs = InFileArgs.fromValues({});
+
+final tests = [
+  _InFileTestArgsTest(
+    name: 'empty',
+    input: '',
+    output: InFileArgs.fromValues({
+      InFileArgItems.experimentsOn: _defaultArgs.experimentsOn,
+      InFileArgItems.appPath: _defaultArgs.appPath,
+    }),
+  ),
+  _InFileTestArgsTest(
+    name: 'non-empty',
+    input: '''
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Do not delete these arguments. They are parsed by test runner.
+//test-argument:appPath="$_testAppPath"
+//   test-argument    :   experimentsOn    =    true
+
+
+import 'dart:ui' as ui;
+''',
+    output: InFileArgs.fromValues({
+      InFileArgItems.experimentsOn: true,
+      InFileArgItems.appPath: _testAppPath,
+    }),
+  ),
+];
+
+void main() {
+  for (final t in tests) {
```

Comment:
I don't think this test will actually get run on the bots. I would be surprised since it is in the integration_test folder. We can move the file args parsing code and its test to devtools_test, and then run the dart test on the bots from there.

This also isn't an "integration test" so it should probably be moved under a "test" folder anyway (e.g. devtools_test/test). After moving it to devtools_test, you'll need to tweak the 'main' job in bots.sh to test this directory like we do for devtools_shared

Reply:
Why not? 
All tests under `packages/devtools_app/test` run, and this one will too.

Reply:
Renamed folder to `integration_test_infra` to avoid confusing.

Reply:
This file isn't under devtools_app/test, so it is not covered by that use case

Reply:
New location sgtm, disregard my last comment.